Tools/scripts/extract_features.py

```diff
@@ -242,6 +242,7 @@ def __init__(self, filename, nm="arm-none-eabi-nm", strings="strings"):
             ('FORCE_APJ_DEFAULT_PARAMETERS', 'AP_Param::param_defaults_data'),
             ('HAL_BUTTON_ENABLED', 'AP_Button::update'),
             ('HAL_LOGGING_ENABLED', 'AP_Logger::Init'),
+            ('AP_OSD_CRSF_EXTENSIONS_ENABLED', r'AP_OSD_Screen::draw_crsf_tx_power'),
```

Comment:
This name is different to the PANELS name - why is that?

Reply:
Seems like you have two defines - they both should go in here

Reply:
Yes, there are two defines. The AP_OSD_CRSF_EXTENSIONS_ENABLED is supposed to be the "global" one, that can be combined with other "globals" to enable the panels themselves via a "local" define (the AP_OSD_CRSF_PANELS_ENABLED). On AP_OSD.h, line 55 [here](https://github.com/ArduPilot/ardupilot/pull/25844/files#diff-2d27ad6b9c73dbb0c2e46daf08ef2b34117a0665902d9115d1185d53e61e52aeR55) is where the PANELS define is defined and it's value set based on both globals that it should be dependent on. 

I decided to do it this way (with the two defines) to make it easier to manage multiple dependencies, if needed.